cookbook/network_from_orion_fepp.ipynb

```diff
@@ -15,30 +15,7 @@
    "source": [
```

Comment:
<p>hyperlink render isn't working on reviewnb, does it work for you locally?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/275/discussion/#comment-2608491015'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>